src/validators/isNumber.js

```diff
@@ -1,13 +1,9 @@
-import {unit} from 'postcss-value-parser';
 import endsWith from 'ends-with';
 
-export default ({type, value}) => {
-    if (type !== 'word') {
+export default (node) => {
```

Comment:
@ben-eb in the future we'll need to check if number is `calc` function. `isCalc() `consumes `node` type only